src/gmpints.c

```diff
@@ -2176,6 +2176,42 @@ Obj FuncPOWERMODINT ( Obj self, Obj base, Obj exp, Obj mod )
 
 /****************************************************************************
 **
+*/
+Obj IsProbablyPrimeInt ( Obj n, Int reps )
+{
+  fake_mpz_t n_mpz;
+  Int res;
+
+  if ( reps < 1 )
+    ErrorMayQuit( "IsProbablyPrimeInt: <reps> must be positive", 0L, 0L );
+
+  CHECK_INT(n);
+
+  FAKEMPZ_GMPorINTOBJ( n_mpz, n );
+
+  res = mpz_probab_prime_p( MPZ_FAKEMPZ(n_mpz), reps );
+
+  if (res == 2) return True; /* definitely prime */
+  if (res == 0) return False; /* definitely not prime */
+  return Fail; /* probably prime */
+}
+
+/****************************************************************************
+**
+*/
+Obj FuncIS_PROBAB_PRIME_INT ( Obj self, Obj n, Obj reps )
+{
+  REQUIRE_INT_ARG( "IsProbablyPrimeInt", "n", n );
+  REQUIRE_INT_ARG( "IsProbablyPrimeInt", "reps", reps );
+  if ( ! IS_INTOBJ(reps) )
```

Comment:
While not required for this patch, I think a `REQUIRE_SMALL_INT_ARG` might be a nice addition.